### ISSUE TITLE
Notifications for Interactions on Jetpack sites

### ISSUE THREAD

**Issue by obenland** [Automattic/wordpress-activitypub#1059]
Get a WPcom notification for like and repost. Already works for follows.

**Comment by obenland** [comment]
@mattwiebe I see likes and reposts in `build_activitypub_interaction_note()`—is it a matter of them currently not working and needing to be debugged? Or is there something else missing?

**Comment by mattwiebe** [comment]
> [@mattwiebe](https://github.com/mattwiebe) I see likes and reposts in `build_activitypub_interaction_note()`—is it a matter of them currently not working and needing to be debugged? Or is there something else missing?

I don't think that they work for WoA sites. But maybe they do? That would be awesome. `build_activitypub_interaction_note` works for Simple sites.

**Comment by obenland** [comment]
oh, I see! In that case this is not on the critical path for "Likes and Reposts on Simple"
